### PR TITLE
Add fix for 3.10

### PR DESCRIPTION
**Pull request by ire4ever1190**
Running mcpi with python 3.10 gives this error since the collections modules has changed
```
AttributeError: module 'collections' has no attribute 'Iterable'
```
This PR adds a fix so it continues to work for 3.10

### CODE REVIEW

File: mcpi/util.py [hunk 1]
```diff
@@ -1,4 +1,7 @@
-import collections
+try:
+    import collections.abc as collections
```
Comment (darkmatter2222) [inline mcpi/util.py:2]: What does the "abc" mean? What is that naming convention, I have seen it in other places b4.

Comment (ire4ever1190) [reply mcpi/util.py:2]: it means "abstract base class", more info [here](https://docs.python.org/3/glossary.html#term-abstract-base-class)

**Comment by nobody5050** [comment]
please merge! While I *can* make a virtual environment for mcpi scripts, I'd prefer not to

**Comment by martinohanlon** [comment]
Before I do a bunch of testing, have you tested on previous Python versions?

**Comment by ire4ever1190** [comment]
Yes, worked on 3.9 and 3.10

**Comment by martinohanlon** [comment]
Merging into dev branch to test

**Comment by martinohanlon** [comment]
Thanks for reporting and providing the PR. Merged into master, released v1.2.1 to pypi. You can see the change in #22

**Pull request merged**